README.md

```diff
@@ -1 +1,145 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone Project] <a name="about-project"></a>
+
+
+**Capstone Project** is a personal projects which i have made during microverse studies at the end of the first module
+
+## 🛠 Built With <a name="built-with"></a>
+
+    .Webhint
+    .Styleint
+    .html
+    .css
+
+### Tech Stack <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **adding some emojis to the html file.**
+- **adding some more interesting background to webpage**
+- **adding more semantic tags to make it rwd**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+
+In order to run this project you need: have basic knowledge of html and css.
+
+### Install
+
+Install this project with: vscode, eslint, node_js
+
+### Setup
+
+1) open your local terminal
+2) run git clone https://github.com/sam2if/Capstone_project.git
+
+
+
+### Usage
+
+To run the project, execute the following command: https://github.com/sam2if/Capstone_project.git
+
+
+### Run tests
+
+npx eslint .
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Deployment
+
+You can deploy this project using: github-pages
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [samuel_lemma](https://github.com/sam2if)
+- Twitter: [samuel_lemma](https://twitter.com/samuellemma3733)
+- LinkedIn: [samuel_lemma](https://www.linkedin.com/in/samuel-lemma-7479b1214/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **I will modify the style.css with more interesting background**
+- [ ] **I will add javascript code to add some function to buttons**
+- [ ] **I will add some emojis to make it very simple**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a star!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Cindy Shin in Behance. for the design.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] [REQUIRED] Your `MIT` license link is not working, kindly fix that 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/225590217-6ff781a9-8b61-4354-9446-82b1bfc42149.png)